### ISSUE TITLE
Seeing Errors when Env Variable is set to SPLUNK_LAMBDA_LOCAL_COLLECTOR_ENABLED=false

### ISSUE THREAD

**Issue by storl0rd** [signalfx/splunk-otel-lambda#285]
Exception while exporting metrics HTTPConnectionPool(host='localhost', port=4318): Max retries exceeded with url: /v1/metrics (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f527b1b8590>: Failed to establish a new connection: [Errno 111] Connection refused')) 

We are seeing this error on our Lambda functions instrumented using separate layers for apm and metrics 
arn:aws:lambda:ca-central-1:254067382080:layer:splunk-apm-python:8
arn:aws:lambda:ca-central-1:254067382080:layer:splunk-lambda-metrics:8

They are configured to send the data to a collector running in gateway mode on an EC2 instance. 
The local collector has been disabled by SPLUNK_LAMBDA_LOCAL_COLLECTOR_ENABLED=false ,  however the logs indicate that the instrumentation is still trying to send metrics to 'localhost'

**Comment by aamielsan** [comment]
Maybe a long shot, but we encountered a similar issue with the layer for Java. Fixed it by adding an environment variable `OTEL_EXPORTER_OTLP_ENDPOINT: http://${otelCollectorEndpoint}:4318` because of the [autoconfigure library for Java](https://opentelemetry.io/docs/languages/java/configuration/#zero-code-sdk-autoconfigure). You could also try to use `OTEL_EXPORTER_OTLP_METRICS_ENDPOINT: http://${otelCollectorEndpoint}:4318/v1/metrics`

**Comment by storl0rd** [comment]
> Maybe a long shot, but we encountered a similar issue with the layer for Java. Fixed it by adding an environment variable `OTEL_EXPORTER_OTLP_ENDPOINT: http://${otelCollectorEndpoint}:4318` because of the [autoconfigure library for Java](https://opentelemetry.io/docs/languages/java/configuration/#zero-code-sdk-autoconfigure). You could also try to use `OTEL_EXPORTER_OTLP_METRICS_ENDPOINT: http://${otelCollectorEndpoint}:4318/v1/metrics`

This is what I ended up doing, adding additional variables ensured that the error did not show up.

**Comment by storl0rd** [comment]
> I'd recommend following the directions in https://docs.splunk.com/observability/en/gdi/get-data-in/serverless/aws/otel-lambda-layer/instrumentation/lambda-ec2-collector-gateway.html - you'll save on startup and disk space overhead by not installing the lambda-local collector at all.

Yes in our configuration , I am not sending the metrics to a gateway collector running on EC2, thanks for the response.
